src/Weft/Server.hs

```diff
@@ -65,17 +73,17 @@ server :: (Wefty q)
        -> Gql q m s 'Resolver
        -> IO ()
 server s r = runSettings
-    (appEndo (foldMap Endo s) defaultSettings)
-    (cors extremelyPermissiveCorsPolicy $ app r)
+  (appEndo (foldMap Endo s) defaultSettings)
+  (cors extremelyPermissiveCorsPolicy $ app r)
 
 -- TODO(Jonathan): At some point you should make this less permissive
 extremelyPermissiveCorsPolicy :: Request -> Maybe CorsResourcePolicy
 extremelyPermissiveCorsPolicy _ = Just $ CorsResourcePolicy
-    { corsOrigins           = Nothing
-    , corsMethods           = [methodGet, methodPost, methodOptions]
-    , corsRequestHeaders    = [hContentType]
-    , corsExposedHeaders    = Nothing
-    , corsMaxAge            = Nothing
-    , corsVaryOrigin        = False
-    , corsRequireOrigin     = False
-    , corsIgnoreFailures    = True }
+  { corsOrigins           = Nothing
+  , corsMethods           = [methodGet, methodPost, methodOptions]
+  , corsRequestHeaders    = [hContentType]
+  , corsExposedHeaders    = Nothing
+  , corsMaxAge            = Nothing
+  , corsVaryOrigin        = False
+  , corsRequireOrigin     = False
+  , corsIgnoreFailures    = True }
```

Comment:
nit: the point of leading commas is so that adding to a record/list  doesn't have any line noise in a diff. putting `}` on the same line breaks that :)